src/Sets/Zonotope.jl

```diff
@@ -492,3 +492,44 @@ function quadratic_map(Q::Vector{MT}, Z::Zonotope{N}) where {N, MT<:AbstractMatr
     end
     return Zonotope(d, remove_zero_columns(h))
 end
+
+# ====================================
+# Zonotope vertex enumeration methods
+# ====================================
+
+function _vertices_list_2D(c::AbstractVector{N}, G::AbstractMatrix{N}) where {N}
```

Comment:
I'll take this template, and replace the `same_sign` with `[0, 360)` angle based partition logic.